routers/org/teams.go

```diff
@@ -76,6 +77,9 @@ func TeamsAction(ctx *context.Context) {
 			return
 		}
 		uname := ctx.Query("uname")
+		if strings.Contains(uname, "(") && strings.HasSuffix(uname, ")") {
```

Comment:
I'm confused as to why this helps. If `uname` has parentheses, then it is not a valid username, so `GetUserByName(name)` in line 84 will just return a `ErrUserNotExist`, right?

Reply:
@ethantkoenig Yes, that's what #2973 described.

Reply:
Because if some user has set `FullName`, the name submitted will be `loginname (FullName)` but not `loginname`, so it will return ErrUserNotExist. This PR will accept both two kinds requests.

Reply:
Oh I see. Could we please add a comment explaining the situation?

```
// uname may be formatted as "username (fullname)"
```

Reply:
done.